hydra-explorer/web/src/components/TickBox/index.tsx

```diff
@@ -3,18 +3,32 @@
 import { TickState } from "@/app/model"
 import useDataFetcher from "@/hooks/DataFetcher"
 import { useCardanoExplorer } from "@/providers/CardanoExplorer"
-import { useState } from "react"
+import { useMemo, useState } from "react"
+import { useNetworkContext } from "@/providers/NetworkProvider"
 
 const TickBox = () => {
-    const [tick, setTick] = useState<TickState | null>(null)
+    const [ticks, setTicks] = useState<TickState[]>([])
     const [error, setError] = useState<string | null>(null)
 
-    useDataFetcher<TickState>({
-        url: '/tick',
-        setFetchedData: setTick,
+    const { currentNetwork, currentNetworkMagic } = useNetworkContext()
+
+    useDataFetcher<TickState[]>({
+        url: `/ticks`,
+        setFetchedData: setTicks,
         setError,
     })
 
+    const tick: TickState | undefined = useMemo(() => {
+        return ticks.find(
+            (tick) => {
+                const networkSelected = currentNetwork === "mainnet" ? "Mainnet" : "Testnet"
+                return tick.network === networkSelected &&
+                    tick.networkMagic === currentNetworkMagic
```

Comment:
Could: only filter using network magic to not be relying on consistent data from the api.